gstreamer-sdp/src/m_i_k_e_y_message.rs

```diff
@@ -1,20 +1,22 @@
-// This file was generated by gir (https://github.com/gtk-rs/gir @ 7f5a2b5)
-// from gir-files (https://github.com/gtk-rs/gir-files @ ???)
-// DO NOT EDIT
-
-use MIKEYCacheType;
-use MIKEYMapType;
-use MIKEYPRFFunc;
-use MIKEYPayload;
-use MIKEYPayloadType;
-use MIKEYType;
+use std::mem;
+use std::ptr;
+
 use ffi;
 use glib::translate::*;
 use glib_ffi;
 use gobject_ffi;
 use gst;
-use std::mem;
-use std::ptr;
+
+use auto::{
+    MIKEYCacheType,
+    MIKEYMapType,
+    MIKEYPRFFunc,
+    MIKEYPayload,
+    MIKEYPayloadType,
+};
+use m_i_k_e_y_decrypt_info::MIKEYDecryptInfo;
+use m_i_k_e_y_encrypt_info::MIKEYEncryptInfo;
+use m_i_k_e_y_map_s_r_t_p::MIKEYMapSRTP;
 
 glib_wrapper! {
     pub struct MIKEYMessage(Boxed<ffi::GstMIKEYMessage>);
```

Comment:
Instead of moving the whole file, you can keep the autogenerated one and only have the additional manual things here